client/analytics/settings/config.js

```diff
@@ -24,6 +24,13 @@ const defaultOrderStatuses = [
 	'pending',
 	'on-hold',
 ];
+
+const actionableOrderStatuses = Array.isArray(
+	wcSettings.wcAdminSettings.woocommerce_actionable_order_statuses
+)
+	? wcSettings.wcAdminSettings.woocommerce_actionable_order_statuses
+	: [];
+
```

Comment:
We may as well do the same for `wcSettings.wcAdminSettings.woocommerce_excluded_report_order_statuses`.